src/annotator/adder.js

```diff
@@ -193,6 +189,34 @@ export class Adder {
     return { top, left, arrowDirection };
   }
 
+  /**
+   * Find the zIndex of five nearby elements (left-top, left-bottom, middle-center,
+   * right-top, right-bottom) and use the greatest.
+   *
+   * @param {number} left - Horizontal offset from left edge of viewport.
+   * @param {number} top - Vertical offset from top edge of viewport.
+   * @return {number} - greatest zIndex
+   */
+  _findZindex(top, left) {
+    const width = this._width();
+    const height = this._height();
+
+    // Unique elements and ancestors
+    const elements = new Set([
+      ...document.elementsFromPoint(left, top),
+      ...document.elementsFromPoint(left, top + height),
+      ...document.elementsFromPoint(left + width / 2, top + height / 2),
+      ...document.elementsFromPoint(left + width, top),
+      ...document.elementsFromPoint(left + width, top + height),
```

Comment:
`elementsFromPoint` is [only supported](https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/elementsFromPoint) in quite recent versions of browsers. [`elementFromPoint`](https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/elementFromPoint) has better support (eg. Safari < 12). We could also have a simple fallback for browsers that don't support it by returning a fixed z-index value.